### PR TITLE
Set env variable WEBWORKS_VERSION in manifest

### PR DESCRIPTION
**Pull request by jamesjhedley**
Fixes issue blackberry/BB10-WebWorks-Packager#204

### CODE REVIEW

**Comment by jamesjhedley** [comment]
@nukulb `qnx.callExtensionMethod('application.getenv', 'WEBWORKS_VERSION')` can be used to retrieve the webworks version at any time during execution. I have tested this in webworks.js at the very beginning of the file and the value returns as expected.

**Comment by nukulb** [comment]
I can use it in the first line?

**Comment by nukulb** [comment]
then I am good.

**Comment by nukulb** [comment]
have you tested it with multiple webworks apps running at the same time.

**Comment by jamesjhedley** [comment]
Yes. They each have their own version.

**Comment by nukulb** [comment]
this is a hassle to test, I am putting this is in untested, I know this sucks but it needs to be tested in official SCM builds.

File: lib/native-packager.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ var childProcess = require("child_process"),
     logger = require("./logger"),
     localize = require("./localize"),
     pkgrUtils = require("./packager-utils"),
+    webworksInfo = require('../Framework/lib/webworks-info'),
```
Comment (nukulb) [inline lib/native-packager.js:27]: first time introducing a hard dependency sucks but I guess it will have to be.

**Pull request closed without merging**